packages/react/src/hooks/useReceiveTextStream.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { useRoomContext } from '../context';
+import type { TextStreamHandler } from 'livekit-client';
+
+/**
+ * @beta
+ * set up a text stream handler for the given topic
+ * @param topic - the topic to listen to
+ * @param onTextStreamReceived - the handler to call when a text stream is received
+ * @returns a function to unregister the text stream handler
+ */
+export function useReceiveTextStream(topic: string, onTextStreamReceived: TextStreamHandler) {
```

Comment:
maybe these methods would be better called `useHandleTextStream` to more closely align in name with the base feature

Reply:
it's called `useTextStream` now and automatically handles calls from multiple places, so it won't throw